core/EE_Deprecated.core.php

```diff
@@ -1381,4 +1381,102 @@ public static function is_update_available($basename = '')
         );
         return LicenseService::isUpdateAvailable($basename);
     }
+}
+
+add_filter(
+    'FHEE__EventEspressoBatchRequest__JobHandlers__RegistrationsReport__reg_csv_array',
+    'ee_deprecated_registrations_report_csv_legacy_fields',
+    10,
+    2
+);
+/**
+ * Filters the CSV row to make it appear like the old labels (which were "$pretty_name[$field_name]").
+ * @since $VID:$
+ * @param $csv_row_data
+ * @param $reg_row
+ * @return array for CSV row
+ * @throws EE_Error
+ * @throws InvalidArgumentException
+ * @throws \EventEspresso\core\exceptions\InvalidDataTypeException
+ * @throws \EventEspresso\core\exceptions\InvalidInterfaceException
+ */
```

Comment:
add a `@deprecated` php doc tag please.  I do realize that there is other code in this file that doesn't have that tag (which is unfortunate).  The tag helps us with knowing when something was deprecated.